src/graph/Graph.ts

```diff
@@ -39,7 +39,7 @@ interface IBaseGraphApi<N, E> {
   getNodeValue(node: string): N | undefined;
   getEdgeValue(from: string, to: String): E | undefined;
   removeNodeValue(node: string): boolean;
-  removeEdgeByObj(edge: GraphEdge<E>): boolean;
+  removeEdgeByObj(edge: Omit<GraphEdge<E>, 'value'>): boolean;
```

Comment:
I guess we should make it optional instead of removing it.

Reply:
I fixed that by added Optional type. Thanks!